### DISCUSSION TITLE
GHES Cannot View Files Changed - Something Went Wrong

### DISCUSSION THREAD

**Discussion by silvio-dp** [community/community#5396] [Enterprise] [answered]
I'm accessing my corporates enterprise server with the mobile app but am unable to see the diffs in the files changed. When tapping on "files changed" I receive the error "Something went wrong" with a Try Again button.

When looking at the error response it seems to be complaining about `scrubVideo` and `unfurlReferences` not being attributes to `bodyHTML` in the request.  I can intercept the request using Charles proxy and remove these attributes, doing so returns a successful response with the data I expect. 

```
fragment commentFields on Comment {
  __typename
  ...commonCommentFields
  bodyHTML(
    hideCodeBlobs: true
    includeSuggestedChangesId: true
    renderSuggestedChangesAsText: false
    scrubVideo: false
    unfurlReferences: true
  )
}
```

**Comment by candyho** [comment, marked as answer]
Hey there, thanks for reporting this! Would you mind telling us which device (iOS or Android) and which app version are you using?

**Reply by silvio-dp** [reply]
Hi @candyho - Sure, I'm on an iPhone 11 running iOS 14.7. The Github Mobile Version was 1.27.0 when I reported the issue, but it's been an issue for a while. I couldn't tell you what version I originally installed. I also just pulled the latest 1.29.0 and the issue persists.

**Reply by candyho** [reply]
Thanks for sharing the details with us. We've identified the bug and it should be fixed in the next app update (version 1.30.0). In the meanwhile, you could also install our [TestFlight app](https://testflight.apple.com/join/NLskzwi5) to see if the issue still persist. This is our Beta app for pilot users to try out new features including bug fixes.

**Reply by silvio-dp** [reply]
Very nice! I just tried the beta 1.30.0 and it works like a charm. Thank you for the quick turnaround I look forward to the update 👍
